app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
```

Comment:
Why do you have blog here? You do not have Blog.php Model and you are not using Blog for this project.